hack/bootstrap-cluster.sh

```diff
@@ -14,19 +14,24 @@ echo "Installing the OpenShift GitOps operator subscription:"
 kubectl apply -f $ROOT/openshift-gitops/subscription-openshift-gitops.yaml
 
 echo
-echo "Waiting for default project (and namespace) to exist:"
+echo -n "Waiting for default project (and namespace) to exist: "
 while : ; do
-  kubectl get appproject/default -n openshift-gitops && break
```

Comment:
I do not like infinity loops with break, rather `while ! kubectl get appproject/default -n openshift-gitopsa &>/dev/null; do`

Reply:
We can leave it as is for now, your change was essentially the minimal update to the existing logic.

Reply:
Oh, and I tested this morning on a new cluster and worked so if you revised now I'd have to retest :(

Reply:
Ack, sounds good to me.